packages/nimble-blazor/NimbleBlazor/wwwroot/NimbleBlazor.lib.module.js

```diff
@@ -7,133 +7,145 @@
-    Dialog: {
-        show: async function (dialogReference) {
-            const reason = await dialogReference.show();
-            return reason === window.customElements.get('nimble-dialog').UserDismissed;
-        },
-        close: function (dialogReference) {
-            dialogReference.close();
-        }
-    },
-    Drawer: {
-        show: async function (drawerReference) {
-            const reason = await drawerReference.show();
-            return reason === window.customElements.get('nimble-drawer').UserDismissed;
-        },
-        close: function (drawerReference) {
-            drawerReference.close();
-        }
-    },
-    Table: {
-        setData: async function (tableReference, data) {
-            const dataObject = JSON.parse(data);
-            await tableReference.setData(dataObject);
-        },
-        getSelectedRecordIds: async function (tableReference) {
-            return tableReference.getSelectedRecordIds();
-        },
-        setSelectedRecordIds: async function (tableReference, selectedRecordIds) {
-            await tableReference.setSelectedRecordIds(selectedRecordIds);
+window.NimbleBlazor = (window.NimbleBlazor === undefined)
+    ? {
+        calledAfterStarted: false,
+        Dialog: {
+            show: async function (dialogReference) {
+                const reason = await dialogReference.show();
+                return reason === window.customElements.get('nimble-dialog').UserDismissed;
+            },
+            close: function (dialogReference) {
+                dialogReference.close();
+            }
         },
-        checkValidity: function (tableReference) {
-            return tableReference.checkValidity();
+        Drawer: {
+            show: async function (drawerReference) {
+                const reason = await drawerReference.show();
+                return reason === window.customElements.get('nimble-drawer').UserDismissed;
+            },
+            close: function (drawerReference) {
+                drawerReference.close();
+            }
         },
-        getValidity: function (tableReference) {
-            return tableReference.validity;
+        Table: {
+            setData: async function (tableReference, data) {
+                const dataObject = JSON.parse(data);
+                await tableReference.setData(dataObject);
+            },
+            getSelectedRecordIds: async function (tableReference) {
+                return tableReference.getSelectedRecordIds();
+            },
+            setSelectedRecordIds: async function (tableReference, selectedRecordIds) {
+                await tableReference.setSelectedRecordIds(selectedRecordIds);
+            },
+            checkValidity: function (tableReference) {
+                return tableReference.checkValidity();
+            },
+            getValidity: function (tableReference) {
+                return tableReference.validity;
+            }
         }
     }
-};
+    : window.NimbleBlazor;
```

Comment:
This is choosing to silently ignore NimbleBlazor being loaded twice. I think we had discussed being more assertive about that, i.e. throwing an error with information on avoiding that.

Reply:
Not throwing an error, but doing a `console.warn`.